### PR TITLE
Add tracking to ilyushin

### PR DESCRIPTION
**Pull request by adgad**
/cc @wheresrhys @commuterjoy

### CODE REVIEW

File: views/partials/stream/article-list.html [hunk 1]
```diff
@@ -52,13 +52,13 @@
 </ul>
 {{#if @root.flags.pagination.isSwitchedOn}}
 	{{#if pagination}}
-	<div class="o-ft-buttons__pagination stream__pagination">
+	<div class="o-ft-buttons__pagination stream__pagination" data-trackable="{{pagination}}">
 
-		<a class="o-ft-buttons o-ft-buttons--big"{{#if pagination.previous}} href="?q={{q}}&page={{pagination.previous}}"{{else}} href="#" {{/if}}><i class="o-ft-icons-icon o-ft-icons-icon--arrow-left"></i></a>
+		<a class="o-ft-buttons o-ft-buttons--big"{{#if pagination.previous}} href="?q={{q}}&page={{pagination.previous}}"{{else}} href="#" {{/if}} data-trackable="previous-page"><i class="o-ft-icons-icon o-ft-icons-icon--arrow-left"></i></a>
```
Comment (commuterjoy) [inline views/partials/stream/article-list.html]: maybe add the page number - Eg, `data-trackable="previous-page | 2`

File: views/partials/stream/nav.html [hunk 2]
```diff
@@ -15,16 +15,16 @@ <h2 class="mypage__nav__title">Topics you follow</h2>
             <div class="mypage__nav__stream-links">
                 {{#each following}}
                     {{#if count}}
-                        <a class="mypage__nav__stream-link" href="#{{id}}">{{title}}
+                        <a class="mypage__nav__stream-link" href="#{{id}}" data-trackable="stream-link | anchor">{{title}}
```
Comment (commuterjoy) [inline views/partials/stream/nav.html]: We capture the element name anyway

File: views/partials/stream/nav.html [hunk 3]
```diff
@@ -15,16 +15,16 @@ <h2 class="mypage__nav__title">Topics you follow</h2>
...
                         </a>
                     {{else}}
-                        <a class="mypage__nav__stream-link mypage__nav__stream-link--empty" href="/stream/{{taxonomy}}/{{encode title}}">{{title}}
+                        <a class="mypage__nav__stream-link mypage__nav__stream-link--empty" href="/stream/{{taxonomy}}/{{encode title}}" data-trackable="stream-link | goto">{{title}}
                             <span class="mypage__nav__stream-link__info">(go to page)</span>
                         </a>
                     {{/if}}
                 {{/each}}
-                <button class="mypage__nav__stream-links__toggle">view more<i class="icon-arrow-down"></i></button>
+                <button class="mypage__nav__stream-links__toggle" data-trackable="more-stream-links">view more<i class="icon-arrow-down"></i></button>
```
Comment (commuterjoy) [inline views/partials/stream/nav.html]: maybe 'view-more'

**Comment by commuterjoy** [comment]
:+1: good, but see to comments first.

File: views/partials/stream/nav.html (hunk 2 above)
Comment (adgad) [inline views/partials/stream/nav.html]: This was more to distinguish between the two types of link (one that opens the stream if there are no articles, and one that just anchors to the stream within that page). Unless maybe it's worth capturing href?

Comment (commuterjoy) [inline views/partials/stream/nav.html]: ok, leave it then.

**Pull request merged**